<launch>
	<node name="sehirus_conversion_node" pkg="radar_msg_ros_conversion" type="sehirus_conversion" output="screen" />
	<node name="heartbeat_listener" pkg="rostopic" type="rostopic" args="echo /heartbeat" output="screen" />
	<node name="fake_heartbeat_server" pkg="radar_msg_ros_conversion" type="fake_heartbeat_server.sh" output="screen" cwd="node" />
</launch>